### PR TITLE
Update about page and router links

### PR DESCRIPTION
**Pull request by StevenPG**
Added router pieces so dashboard/about page work, placed a card in the about page. Apparently there's something up with a fresh installation of the project. When Vue UI attempts to refresh based on code changes, it blows up on some Vuetify configuration.

Just means I have to rebuild from scratch on every code change, I wanna poke at that with you at some time in the future.

However, this PR at least gives the About page something, and makes routing work.

### CODE REVIEW

File: ui/kubequotas-front/src/main.ts [hunk 1]
```diff
@@ -2,9 +2,12 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import vuetify from './plugins/vuetify';
+import Vuetify from 'vuetify'
 
 Vue.config.productionTip = false
 
+Vue.use(Vuetify)
```
Comment (shayd3) [inline ui/kubequotas-front/src/main.ts]: Is this part needed? I think this is taken care of by `./src/plugins/vuetify.js`

Comment (StevenPG) [reply ui/kubequotas-front/src/main.ts]: I was running into some resolution issues with vuetify typing, specifically: https://github.com/vuetifyjs/vuetify/issues/5944

From doing some research, it seems to happen on environments other than the original dev environment. We can exclude the commit for now, and I can recreate the issue for us to debug in the future. I do think as it is right now, it's a duplicate import.

However, this was the only way for me to get it working on my local env.

Let me know!

Comment (shayd3) [reply ui/kubequotas-front/src/main.ts]: Works for me! Yeah we can hit that issue later on

Comment (StevenPG) [reply ui/kubequotas-front/src/main.ts]: added issue #23 for this

**Pull request merged**